browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2365,14 +2366,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		annotation.setExpanded();
 		annotation.hideMarker();
 		annotation.sectionProperties.annotationMarker = null;
-
-		var replyCountNode = document.getElementById('reply-count-node-' + data.id);
-		if (replyCountNode)
-			replyCountNode.style.display = 'none';
-
-		var arrowSpan = document.getElementById('arrow span ' + data.id);
```

Comment:
this code is removed by mistake I think?

Reply:
no that was intentional.